sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -52,6 +52,7 @@ class MapboxMap internal constructor(
   internal var isStyleLoadInitiated = false
   private val styleObserver = StyleObserver(this, nativeMapWeakRef, nativeObserver, pixelRatio)
   internal var renderHandler: Handler? = null
+  internal var styleLoaded = false
```

Comment:
should it be with private setter?
because in Java `internal` converted into `public`?

Reply:
We need set it in test cases.